### PR TITLE
[TE] rootcause - adjust display range when investigation out of bounds

### PR DESCRIPTION
**Pull request by apucher**
simple adjustment of display range by window expansion, if the investigation period is out of bounds.

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js [hunk 1]
```diff
@@ -31,14 +31,14 @@ export default Component.extend({
   //
   // external (optional)
   //
-  analysisRangeMax: moment().endOf('day'),
+  analysisRangeMax: moment().endOf('day').valueOf() + 1,
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: what does the +1 do here? Seems very arbitrary

Comment (apucher) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: yeah I could also use startOf('day').plus(1, 'day')

Comment (justYves) [reply thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: @apucher .add(1, 'day') is the correct method I believe 
https://momentjs.com/docs/#/manipulating/add/

File: thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js [hunk 2]
```diff
@@ -31,14 +31,14 @@ export default Component.extend({
...
-  analysisRangeMax: moment().endOf('day'),
+  analysisRangeMax: moment().startOf('day').add(1, 'day'),
 
   analysisRangePredefined: {
-    'Today': [moment(), moment().endOf('day')],
-    'Last 3 days': [moment().subtract(2, 'days').startOf('day'), moment().endOf('day')],
-    'Last 7 days': [moment().subtract(6, 'days').startOf('day'), moment().endOf('day')],
-    'Last 14 days': [moment().subtract(13, 'days').startOf('day'), moment().endOf('day')],
-    'Last 28 days': [moment().subtract(27, 'days').startOf('day'), moment().endOf('day')]
+    'Today': [moment(), moment().startOf('day').add(1, 'hour')],
```
Comment (justYves) [inline thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart-toolbar/component.js]: nit: technically it's 'days' and 'hours', but moment handles this as well (not too sure if this is documented though) 

```
moment().add(1, 'days').calendar();  
// from their doc https://momentjs.com/
```

**Pull request merged**